src/Elcodi/Component/Cart/Entity/Order.php

```diff
@@ -106,6 +107,27 @@ class Order implements OrderInterface
     protected $shippingMethod;
 
     /**
+     * @var string
+     *
+     * Shipping method extra
+     */
+    protected $shippingMethodExtra;
```

Comment:
Initialize them as arrays allows you to remove these last two lines: https://github.com/elcodi/elcodi/pull/982/files#diff-9c57f24f614243f7a78090b60f906198R99